packages/node/src/utils/hot-reload.js

```diff
@@ -0,0 +1,77 @@
+const hashmap = {};
```

Comment:
Can we change this file to Typescript?

Reply:
Yeah I was just in a rush and don't know TS well enough to actually convert it

Reply:
So, is this differed?

Reply:
I’d defer it as it’ll take me 10x longer to try and write it in TS lol